envs/monkey_zoo/packer/opensmtpd_68.pkr.hcl

```diff
@@ -0,0 +1,20 @@
+source "googlecompute" "opensmtpd-68" {
+    image_name = "opensmtpd-68"
+    project_id = "${var.project_id}"
+    source_image = "debian-10-buster-v20240515"
+    zone = "${var.zone}"
+    disk_size = 10
+    machine_type = "${var.machine_type}"
+    ssh_username = "vzilius"
```

Comment:
This was the problem. Setting this to `root` caused the failed SSH authentication

Reply:
I've also logged in with gcloud account manually, added permissions to gcp service account. If it doesn't work for you consider changing this to your account name

Reply:
This should be changed to `m0nk3y` user I think.